src/com/pentalog/pentastagiu/homework2_usersMessagesOnBoard/UserMessage.java

```diff
@@ -0,0 +1,38 @@
+package com.pentalog.pentastagiu.homework2_usersMessagesOnBoard;
+
+public class UserMessage {
```

Comment:
e suficient sa denumesti Message, daca nu lucrezi in proiect cu mai multe tipuri de mesaje este destul de clar ce reprezinta